tutorials/users/ide/version-control.md

```diff
@@ -1 +1,614 @@
+
+Currently, there is no mechanism in the GUI to stash files.  Files can only be stashed from the Terminal.  
+Similarly, files can only be unstashed from the Terminal.
+
+[Stash Files From Terminal](content:references:stash-terminal)
+
+Once the command is run all files listed under `Changed` will be moved to the `Stash` and will not be seen. 
+
+
+### Creating a Branch
+
+Click the down arrow in the `Current Branch` section in the Git sidebar.  It will reveal the `Branches` window and show all the created branches.
+
+Click the `New Branch` button.
+
+![New Branch Button](images/version-control/new-branch-button.png)
+
+Fill out the form and input a branch name, and click which branch to base the new branch off of.  After that click `Create Branch`.
+
+It is good practice to not use spaces or special characters in the branch name other than dashes.
+
+
+![New Branch Form](images/version-control/new-branch-dialog.png)
+
+Git will automatically switch you into the new branch.  Your newly created branch will be shown under `Current Branch` and be in the list of branches.  
+
+![Create Branch Successful](images/version-control/create-branch-successful.png)
+
+
+### Switching Branches
+
+```{note}
+You can only switch branches when you have no conflicts in files between the branches.
+```
+
+To switch branches simply click another branch listed under the `Branches` section.
+
+```{note}
+You can only switch to another local branch.  Branches with the `origin` prefix are remote branches and cannot be switched into.
+``` 
+
+
+## Using Git with the Terminal
+
+In this section are some useful Git commands.  For more in-depth information take a look at the 
+[official Git documentation](https://git-scm.com/doc).
+
+### Set Up Your Git Identifier
+
+When you set up your Git Identifier your username and email will be shown with each commit you make.
+If you are working in a team this will show who made a certain commit.
+
+```
+git config --global user.name "Your Name"
+git config --global user.email "email@email.com"
+```
+
+### Staging a File
+
+A file can be staged whether you are in the same directory as the file or not. 
+
+To stage a file when you are in the same directory as the file, you use the command with only the filename.
+
+```
+git add <filename>
+```
+
+To stage a file when you are in a different directory as the file, you use the command with the folder path to the file.
+
+```
+git add /folder/path/to/file/<filename>
+```
+
+### View Status
+
+If you have multiple files staged and ready to commit you can see the list by using `git status`.
+
+```
+git status
+```
+
+
+### Making a Commit
+
+Add a brief descriptive message about your commit.
+
+```
+git commit -m "your brief commit message goes here"
+```
+
+### Pushing a Commit
+
+After your commit is complete, push it to the remote repository.
+
+```
+git push
+```
+
+### Pushing a Commit to an Existing Remote Repository
```

Comment:
This heading should be under "Pushing a Commit" so I'd make it with four `#`s instead of three.

You could also just remove this heading since this information makes sense under the "Pushing a Commit" heading as well

Reply:
Removed the heading